solidity/ecdsa/deploy/02_deploy_reimbursement_pool.ts

```diff
@@ -3,11 +3,11 @@ import type { HardhatRuntimeEnvironment } from "hardhat/types"
 import type { DeployFunction } from "hardhat-deploy/types"
 
 const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
-  const { getNamedAccounts, deployments } = hre
+  const { getNamedAccounts, ethers, deployments } = hre
   const { deployer } = await getNamedAccounts()
 
-  const staticGas = 40800 // gas amount consumed by the refund() + tx cost
-  const maxGasPrice = 500000000000 // 500 gwei
+  const staticGas = 48_000 // gas amount consumed by the refund() + tx cost
```

Comment:
Why change to 48k? `40800` was chosen as an average gas amount for the `ReimbursementPool.refund()` function. If we run the tests in `ReimbursmentPool.test.ts` and look at the returned transaction `gasUsed` from the `refund()` then the gas used is between 40433 - 41893. The higher-end is when on the rare occasion something goes south with the refund. Having said that, 48k seems to be too high. We want callers to break even as much as we can or make them just slightly net-positive, but not incentivize them with additional ETH from the Reimbursement pool.
Also, changing it to 48k. breaks around 10 tests in the `random-beacon`. We need to be careful with this number, cause we have one Reimbursement pool for `random-beacon`, `ecdsa` and possibly for other contracts like maintainer proxy.
I think we should leave it as-is and tweak the gas offsets instead.

Reply:
I changed the value as we've got a very simple function `notifySeedTimeout` that uses `refundable` modifier:
https://github.com/keep-network/keep-core/blob/a98a1e4d38c3c4c3093284663f6a650101a54a0f/solidity/ecdsa/contracts/WalletRegistry.sol#L688-L690

If I leave `staticGas = 40800` a test for refund fails as the submitter's balance is negative it requires bumping the static gas up to around 4800 to let the test pass.

`notifySeedTimeout` is a very basic function, does it make sense to add an offset to it? Calling the function via proxy probably added the ~7k gas. If we don't need to modify the `staticGas = 40800`, then we would need to add offset to each function called in an upgradable contract.

Reply:
> Calling the function via proxy probably added the ~7k gas

ok, now it adds up. This is specific for `WalletRegistry`, and `ReimbursementPool` is an independent contract that doesn't care about the details of the callee. It will only reimburse for the gas that it consumes, and if a caller contract uses more gas on top, then it should be passed to the `refund` function. This way any contract can count on `ReimbursementPool` for the almost exact refund.

Reply:
Please see b5a155457096170982f028e0d683cbd2eb2c9973